test/Preprocessing/lit.local.cfg.py

```diff
@@ -0,0 +1,16 @@
+# -*- Python -*-
+
+import os
+import platform
+import sys
+
+import lit.formats
+from lit.llvm import llvm_config
+
+# Configuration file for the 'lit' test runner.
+
+# testFormat: The test format to use to interpret tests.
+config.test_format = lit.formats.ShTest(execute_external=False)
+
```

Comment:
Correct me if I am wrong, but I don't think anything above this line is needed for the intended purpose of this file which is simply to hide these potential test input files from lit's search algorithms. All you need is the line setting `config.suffixes`